database/schema.sql

```diff
@@ -853,6 +853,8 @@ ALTER TABLE dbo.sccv_person_case_status_answers
   created_at timestamp,
   referral_doc_url varchar(256),
   stage varchar(30),
+  contact_decision_created_at timestamp,
```

Comment:
to do:
1- alter table statement here
2- alter table statement in separate file in manual updates to track the actual intervention on the db